energy.md

```diff
@@ -125,28 +129,30 @@ In energy transformation sectors, the output unit and input unit are EJ (per yea
 In the schematic of the energy system depicted below, the energy transformation and distribution sectors include all sectors except for the resources (colored red) and the final demands (colored light blue).
 
 ![Figure 3](gcam-figs/energy_system_structure.png)<br/>
-Figure 3: Simplified schematic of the energy system in each region, showing the inter-sectoral flows of energy goods in GCAM.
+Simplified schematic of the energy system in each region, showing the inter-sectoral flows of energy goods in GCAM.
 {: .fig}
 
 
 ### Electricity
 
-*Under Construction*
+The GCAM electricity sector models the conversion of primary fuels (e.g., coal, gas, oil, bioenergy) to electricity. For most fuels, GCAM includes several different technology options (e.g., pulverized coal, coal IGCC, etc.). Individual technologies compete for market share based on their technological characteristics (conversion efficiency in the production of products from inputs), and cost of inputs and price of outputs. The cost of a technology in any period depends on (1) its exogenously specified non-energy cost, (2) its endogenously calculated fuel cost, and (3) any cost of emissions, as determined by the climate policy. The first term, non-energy cost, represents capital, fixed and variable O&M costs incurred over the lifetime of the equipment (except for fuel or electricity costs). For electricity technologies, GCAM read in each of these terms and compute the levelized cost of energy within the model. For example, the non-energy cost of coal-fired power plant is calculated as the sum of overnight capital cost (amortized using a capital recovery factor and converted to dollars per unit of energy output by applying a capacity factor), fixed and variable operations and maintenance costs. The second term, fuel or electricity cost, depends on the specified efficiency of the technology, which determines the amount of fuel or electricity required to produce each unit of output, as well as the cost of the fuel or electricity.
```

Comment:
Subject-verb agreement problem here:
> GCAM read in each of these terms and compute ...

BTW, you can wrap these lines in the markdown file, and the display output will be the same.  It takes two blank lines to make a paragraph break.  I encourage people to do that, both because it makes it easier to point out small issues like this (you can put the comment on the line), and because it makes the version history more useful (an edit shows up as an edit to the individual line, instead of the whole paragraph.